src/runtime/pkg/resourcecontrol/utils_linux.go

```diff
@@ -43,28 +45,36 @@ func ValidCgroupPathV1(path string, systemdCgroup bool) (string, error) {
 	return filepath.Join(DefaultResourceControllerID, filepath.Clean("/"+path)), nil
 }
 
-// ValidCgroupPathV2 returns a valid cgroup path for cgroup v2.
-// see https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#cgroups-path
-func ValidCgroupPathV2(path string, systemdCgroup bool) (string, error) {
+func AbsolutePath(path string) string {
```

Comment:
Are `AbsolutePath` and `RandStr` need to be package exported names?

Reply:
fixed